app/views/movies/_index.html.erb

```diff
@@ -1,49 +1,57 @@
 <p id="notice"><%= notice %></p>
 
 <h1>List of Movies in the DB</h1>
-<%= link_to "Sort by rating (unrated not included)", movies_rating_url %> | <%= link_to "Show all unrated films", movies_unrated_url %>
-| <%= link_to "Sort by Year (latest first)", movies_latest_url %>
+<%= link_to "Sort by rating (unrated not included)", sort_movies_path(rating: true) %> | <%= link_to "Show all unrated films", sort_movies_path(unrated: true) %>
+| <%= link_to "Sort by Year (latest first)", sort_movies_path(year: true) %>
 <% if @movies.count > 0%>
-<%# Create a table for movie iteration %>
-<table class="table table-hover">
-  <thead>
-    <tr>
-      <th>Title</th>
-      <th>Year</th>
-      <th>Synopsis</th>
-      <th>Rating</th>
-      <th>No. of Reviews</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+  <%# Create a table for movie iteration %>
```

Comment:
Please code with proper alignment, it help a lot to know what the code doing, editor can help just tap tap.